symplyphysics/laws/hydro/froude_number.py

```diff
@@ -0,0 +1,44 @@
+from sympy import Eq, solve, sqrt, S
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless, convert_to)
+
+
+# Description
+# Froude number characterizes the ratio between the force of inertia and
+# the external force, in the field of which the motion occurs, acting on
+# an elementary volume of liquid or gas.
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length (The characteristic length is the dimension
+# that defines the length scale of a physical system. A characteristic length
+# is usually the volume of a system divided by its surface: L = V / A,
+# where V is the volume of the body, and A is the cross-sectional area.
+# For example, it is used to calculate flow through circular and non-circular
+# tubes in order to examine flow conditions. D = 4 * A / p, where
+# D is characteristic diameter, A is the cross-sectional are, p is wetted perimeter),
```

Comment:
Still not perfect, but let's keep it this way. Please move it to description, to avoid cluttering the law.